test/factories/users.rb

```diff
@@ -12,7 +12,7 @@
     bio { Faker::Company.bs }
     timezone 'EST'
     mentor false
-    verified false
+    verified true
```

Comment:
Let's keep this set to `false` by default, as this is not the initial sign on case until someone goes through the ID.me process.

If you have need it to be `true`, you can either:

- use the `trait`:  https://github.com/OperationCode/operationcode_backend/blob/master/test/factories/users.rb#L26-L28
- set it to `true` in the test in question